### PR TITLE
:sparkles: feat: Create Reusable InfoCard Component #28

### PR DESCRIPTION
**Pull request by thadeusdev**
infoCard

# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code

- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: libs/elements/cards/src/lib/cards.routing.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core'
+import { RouterModule, Routes } from '@angular/router'
+import { ElewaInfoCardComponent } from './components/elewa-info-card/elewa-info-card.component';
+
+export const CARDS_ROUTES: Routes = [
+    { path: '', component: ElewaInfoCardComponent}
+];
+
```
Comment (LemmyMwaura) [inline libs/elements/cards/src/lib/cards.routing.ts:8]: No need for routing on this module, we don't have a root path that renders or links to this module

File: libs/elements/cards/src/lib/material.module.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { NgModule } from "@angular/core";
+import { MatIconModule } from '@angular/material/icon'
+
```
Comment (LemmyMwaura) [inline libs/elements/cards/src/lib/material.module.ts:3]: This shouldn't be here. But I'll set this up on the project. Check for updates in a few

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Very clean work. Just clean up a few things

File: apps/elewa-website/src/app/app.module.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { AppComponent } from './app.component';
 import { TranslocoRootModule } from './transloco-root.module';
 
 import { AppRoutingModule } from './app.routing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```
[the remark this answers is not in the document]
Comment (thadeusdev) [reply apps/elewa-website/src/app/app.module.ts]: I have fixed this part. Thank you for the heads up

File: libs/elements/cards/src/lib/cards.routing.ts (hunk 1 above)
Comment (thadeusdev) [reply libs/elements/cards/src/lib/cards.routing.ts]: I have fixed this one too. Thank you for the heads up

File: libs/elements/cards/src/lib/material.module.ts (hunk 2 above)
Comment (LemmyMwaura) [reply libs/elements/cards/src/lib/material.module.ts]: Anytime

**Pull request closed without merging**